Scripts/DCS-BIOS/lib/modules/aircraft_modules/F-15E.lua

```diff
@@ -0,0 +1,1315 @@
+module("F-15E", package.seeall)
+
+local Control = require("Control")
+local ControlType = require("ControlType")
+local Functions = require("Functions")
+local IntegerOutput = require("IntegerOutput")
+local Module = require("Module")
+local PhysicalVariant = require("PhysicalVariant")
+local Suffix = require("Suffix")
+
+local Module = require("Module")
+
+--- @class F_15: Module
+local F_15 = Module:new("F-15E", 0x9200, { "F-15ESE" })
+
+--by WarLord,Maverick87Shaka&RafaPolit v1.2c
+
+--UFC DISPLAY (ID: 8)
+local F_UFC_LINE1 = ""
+local F_UFC_LINE2 = ""
+local F_UFC_LINE3 = ""
+local F_UFC_LINE4 = ""
+local F_UFC_LINE5 = ""
+local F_UFC_LINE6 = ""
+local F_UFC_LINE1_dots = ""
+local F_UFC_LINE2_dots = ""
+local F_UFC_LINE3_dots = ""
+local F_UFC_LINE4_dots = ""
+local F_UFC_LINE5_dots = ""
+local F_UFC_LINE6_dots = ""
+local F_UFC_ActiveUHF1 = ""
+local F_UFC_ActiveUHF2 = ""
+
+--REAR SEAT UFC DISPLAY (ID:20)
+local R_UFC_LINE1 = ""
+local R_UFC_LINE2 = ""
+local R_UFC_LINE3 = ""
+local R_UFC_LINE4 = ""
+local R_UFC_LINE5 = ""
+local R_UFC_LINE6 = ""
+local R_UFC_LINE1_dots = ""
+local R_UFC_LINE2_dots = ""
+local R_UFC_LINE3_dots = ""
+local R_UFC_LINE4_dots = ""
+local R_UFC_LINE5_dots = ""
+local R_UFC_LINE6_dots = ""
+local R_UFC_ActiveUHF1 = ""
+local R_UFC_ActiveUHF2 = ""
+
+local ufc_string_length = 20
+
+-- remove Arg# Pilot 1020 / WSO 1021 / WSO INSTR 1039
+
+--- Defines a light
+--- @param self Module the calling module
+--- @param identifier string the unique identifier for the control
+--- @param arg_number integer the dcs argument number
+--- @param category string the category in which the control should appear
+--- @param description string additional information about the control
+--- @return Control control the control which was added to the module
+local function defineIndicatorLightMulti1(self, identifier, arg_number, category, description) --red
+	local max_value = 1
+	local alloc = self:allocateInt(max_value)
+
+	self:addExportHook(function(dev0)
+		if dev0:get_argument_value(arg_number) >= 0.1 and dev0:get_argument_value(arg_number) < 0.6 then
+			alloc:setValue(1)
+		else
+			alloc:setValue(0)
+		end
+	end)
+
+	local control = Control:new(category, ControlType.led, identifier, description, {}, {
+		IntegerOutput:new(alloc, Suffix.none, "Multi Led Color 1; Light is on between 0.1 and 0.59"),
+	})
+	self:addControl(control)
+
+	return control
+end
+
+--- Defines a light
+--- @param self Module the calling module
+--- @param identifier string the unique identifier for the control
+--- @param arg_number integer the dcs argument number
+--- @param category string the category in which the control should appear
+--- @param description string additional information about the control
+--- @return Control control the control which was added to the module
+local function defineIndicatorLightMulti2(self, identifier, arg_number, category, description) --green
```

Comment:
This can also be replaced with defineGatedIndicatorLight

Reply:
done